src/main/resources/benchmarks/auctionmark/ddl-generic.sql

```diff
@@ -23,7 +23,7 @@ CREATE TABLE config_profile (
                                 cfp_scale_factor            FLOAT NOT NULL,
                                 cfp_loader_start            TIMESTAMP NOT NULL,
                                 cfp_loader_stop             TIMESTAMP NOT NULL,
-                                cfp_user_item_histogram     VARCHAR(12000) NOT NULL
+                                cfp_user_item_histogram     LONGVARCHAR NOT NULL
```

Comment:
We should use `TEXT` for the generic schema. That's more common in DBMSs than `LONGVARCHAR`.

Reply:
so my understanding is that `generic` ddl is used for creating the hsqldb schema.  i took a quick look at the hsqldb docs and didn't see a `STRING` or `TEXT` data type and that `LONGVARCHAR` was the equivalent.  I used `TEXT` for the other flavors of `ddl-*.sql`.

Reply:
Hey Andy any more thoughts here?

Reply:
I think LONGVARCHAR is going to break most systems. It looks like we should run HSQLDB in MySQL compatibility mode:

https://stackoverflow.com/a/12290408

I won't have time to test this today.